src/cli.js

```diff
@@ -28,6 +28,10 @@ program
     '-d, --out-dir <path>',
     'set the output directory. defaults to "dist"'
   )
+  .option(
+    '-f, --out-file <filename>',
```

Comment:
Can we make the alias `-o` instead? I see it's already aliased to `--open`, but IMO that doesn't need an alias and `-o` should alias to `--out-file` instead.